src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -120,7 +120,7 @@ class BeamMobsim @Inject()(
           "RideHailManager"
         )
         context.watch(rideHailManager)
-        Await.result(rideHailManager ? Identify(0), timeout.duration)
+//        Await.result(rideHailManager ? Identify(0), timeout.duration)
```

Comment:
Another area I wanted to make sure isn't a leftover from testing. I don't see any changes in RHM which would mean we no longer need to wait